worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java

```diff
@@ -156,15 +156,8 @@ public void thru(Player player) throws WorldEditException {
         desc = "Teleport to a location"
     )
     @CommandPermissions("worldedit.navigation.jumpto.command")
-    public void jumpTo(Player player,
-        @Arg(desc = "Location to jump to", def = "")
-            Location pos,
-        @Switch(name = 'f', desc = "force teleport")
-            boolean force) throws WorldEditException {
-
-        if (pos == null) {
-            pos = player.getSolidBlockTrace(300);
-        }
+    public void jumpTo(Player player) throws WorldEditException {
+        Location pos = player.getSolidBlockTrace(300);
```

Comment:
why are these removed?

Reply:
I'm not aware of any scenario where the place needs to enforce a teleport on themself on a given location.